src/runtime/virtcontainers/qemu.go

```diff
@@ -2203,7 +2203,7 @@ func (q *qemu) getThreadIDs(ctx context.Context) (vcpuThreadIDs, error) {
 	tid.vcpus = make(map[int]int, len(cpuInfos))
 	for _, i := range cpuInfos {
 		if i.ThreadID > 0 {
-			tid.vcpus[i.CPU] = i.ThreadID
+			tid.vcpus[i.CPUIndex] = i.ThreadID
```

Comment:
So, the CPU field got renamed to CPUIndex, is it?

Reply:
Not exactly.  It's actually a entirely different structure, though it has similar contents.  The only change that mattered to us was that cpu index is how in this field rather than the other one.

Reply:
Ack.